components/NewRealmWizard/components/steps/InviteMembersForm.tsx

```diff
@@ -131,6 +135,58 @@ export interface InviteMembers {
   memberAddresses: string[]
 }
 
+/**
+ * Convert a list of addresses/domains into a list of resolved addresses
+ */
+async function resolveAddressList(connection: Connection, textBlock: string) {
+  const items = textBlock.split(/[\n,]+/).map((item) => item.trim()).filter(Boolean)
+  console.log('Resolving items:', items)
+  
+  const results = await Promise.all(
+    items.map(async (item) => {
+      try {
+        // If it's already a valid address, return it
+        if (validateSolAddress(item)) {
+          console.log(`${item} is already a valid address`)
+          return { input: item, resolved: item }
+        }
+        
+        // Try to resolve as domain
+        console.log(`Attempting to resolve domain: ${item}`)
+        const resolved = await resolveDomain(connection, item)
+        console.log('Resolved:', resolved)
+        console.log(`Domain ${item} resolved to:`, resolved?.toBase58() || 'null')
+        return {
+          input: item,
+          resolved: resolved?.toBase58()
+        }
+      } catch (error) {
+        console.error(`Error resolving ${item}:`, error)
+        return { input: item, resolved: undefined }
+      }
+    })
+  )
+
+  const valid: string[] = []
+  const invalid: string[] = []
+
+  results.forEach(({ input, resolved }) => {
+    if (resolved && validateSolAddress(resolved)) {
```

Comment:
can you please explain why is `validateSolAddress()` needed here?

Reply:
the function was already present in the repo, here it's just a safety check here to avoid the `resolveDomain` function to throw any error

Reply:
results would then only include `validateSolAddress()` if it is true, if it is false then it would ignore it.

Reply:
wait, I understand now. you can ignore me.